OHHTTPStubs/UnitTests/Test Suites/SwiftHelpersTests.swift

```diff
@@ -52,6 +52,28 @@ class SwiftHelpersTests : XCTestCase {
     }
   }
 
+  func testIsAbsoluteUrl() {
+    let matcher = isAbsoluteUrl("foo://foo/bar")
+
+    let urls = [
+      "foo:": false,
+      "foo://foo/bar/baz": false,
+      "foo://foo/bar": true,
+      "bar://foo/bar": false,
```

Comment:
Maybe add one or two more complex examples, I'm especially thinking about URLs with query parameters and anchors etc